src/lesson7/Animal.java

```diff
@@ -0,0 +1,45 @@
+package lesson7;
+
+import java.util.Objects;
+
+
+public abstract class Animal {
+
+    protected final String NAME;
```

Comment:
```suggestion
    protected final String name;
```
только константы static final именуем капсом